assets/ui.css

```diff
@@ -1,3 +1,14 @@
+.close-button{
+  font-weight: bold;
+  float: left;
+  cursor: pointer;
```

Comment:
Not sure this one is needed, @sidvishnoi what do you think?

Reply:
float:left would be fine. I think.

Reply:
Yeah, I mean `cursor: pointer` 👀

Reply:
Also, is this margin for the close button fine or i shall decrease it
![Screenshot from 2020-01-09 13-48-19](https://user-images.githubusercontent.com/44131389/72050302-ce1b8b80-32e6-11ea-93b8-b848f1a17a0f.png)

Reply:
IMO it's okay as-is.

Reply:
I prefer to keep `cursor:pointer` and remove the scaling animation.